### PR TITLE
feed and addpost pages

### PR DESCRIPTION
**Pull request by romanpylypyak**

### CODE REVIEW

File: client/src/layouts/Header.vue [hunk 1]
```diff
@@ -1,67 +1,97 @@
...
-  >
-    <q-toolbar>
-      <router-link
-        :to="{path: '/'}"
-        class="homeRouterLink"
-      >
-        <img
-          src="@/assets/img/logo.jpeg"
-          class="q-ma-sm logo"
+  <div class='header' @click='hideSearch'>
```
Comment (igorkokotko) [inline client/src/layouts/Header.vue]: Please use double-quotes.

File: client/src/layouts/Header.vue [hunk 2]
```diff
@@ -1,67 +1,97 @@
...
+        <q-btn
+          flat
+          round
+          dense
+          icon="menu"
+          class="text-cyan"
+        >
+          <q-menu>
+            <q-list style="min-width: 100px">
+              <template v-if="$store.state.auth.token">
```
Comment (igorkokotko) [inline client/src/layouts/Header.vue]: Use Getters - https://vuex.vuejs.org/guide/getters.html#method-style-access

File: client/src/layouts/Header.vue [hunk 3]
```diff
@@ -1,67 +1,97 @@
...
+    return {
+      search: ''
+    }
+  },
+  methods: {
+    showSearch () {
+      this.$refs.search.style.visibility = 'visible'
+    },
+    hideSearch (e) {
+      if (e.target.className === 'q-toolbar row no-wrap items-center') {
```
Comment (igorkokotko) [inline client/src/layouts/Header.vue]: Please figure out a better approach as sometimes class names might changing.

File: client/src/layouts/Header.vue [hunk 4]
```diff
@@ -1,67 +1,97 @@
...
+        this.$refs.search.style.visibility = 'hidden'
```
Comment (igorkokotko) [inline client/src/layouts/Header.vue:83]: Try to merge showSearch and hideSearch into one function where you just pass 'visibility' as arguments, something like toggleSearchVisibility(visibility)

File: client/src/views/Feed/Feed.vue [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+
+    play: function (event) {
+      let currentVideo = event.target
+      if (currentVideo.paused) {
+        currentVideo.play()
+      } else {
+        currentVideo.pause()
+      }
+    },
+    test () {
```
Comment (igorkokotko) [inline client/src/views/Feed/Feed.vue]: WTF?

File: client/src/views/Feed/AddPost.vue [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+.inputfile{
+display: none;
+}
+#modal-window .q-card__section{
+  height: 70px;
+}
+#modal-window .q-bar{
+  background: #87e0f5
+}
+label p{
```
Comment (igorkokotko) [inline client/src/views/Feed/AddPost.vue]: It's a bad idea to styling globally label p. Use scoped style or add class.

**Review by igorkokotko — CHANGES_REQUESTED** [review]

File: client/src/views/Feed/Feed.vue [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+word-wrap:break-word;
+        span {
+            font-weight: bold;
+        }
+}
+.content .caption2{
+  position: absolute;
+  width: 88%;
+ bottom: 4%;
+font-size: 0.85rem;
```
Comment (de5peratelighter) [inline client/src/views/Feed/Feed.vue]: Something is wrong with identation with this css;)

File: client/src/layouts/Header.vue (hunk 1 above)
Comment (romanpylypyak) [reply client/src/layouts/Header.vue]: use double-quotes in all lines? Or double-quotes in template only?

Comment (romanpylypyak) [reply client/src/layouts/Header.vue]: fixed that line

File: client/src/layouts/Header.vue (hunk 2 above)
Comment (romanpylypyak) [reply client/src/layouts/Header.vue]: add getter in store

File: client/src/layouts/Header.vue (hunk 3 above)
Comment (romanpylypyak) [reply client/src/layouts/Header.vue]: fixed

File: client/src/layouts/Header.vue (hunk 4 above)
Comment (romanpylypyak) [reply client/src/layouts/Header.vue:83]: merge to  one function

File: client/src/views/Feed/Feed.vue (hunk 5 above)
Comment (romanpylypyak) [reply client/src/views/Feed/Feed.vue]: oops, I  delete this function and set route change currently on an element

File: client/src/views/Feed/AddPost.vue (hunk 6 above)
Comment (romanpylypyak) [reply client/src/views/Feed/AddPost.vue]: fixed

File: client/src/views/Feed/Feed.vue (hunk 7 above)
Comment (romanpylypyak) [reply client/src/views/Feed/Feed.vue]: now, indentation I hope looking better

**Review by igorkokotko — APPROVED** [review]

**Pull request merged**